components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthAdminService.java

```diff
@@ -425,6 +425,15 @@ public boolean isPKCESupportEnabled() {
         return oAuthAdminServiceImpl.isPKCESupportEnabled();
     }
 
+    /**
+     * Get the authorization code validity period property value from identity.xml file.
+     *
+     * @return authorization code validity period property value
+     */
+    public long getAuthorizationCodeValidityPeriod() {
+        return oAuthAdminServiceImpl.getAuthorizationCodeValidityPeriod();
```

Comment:
Leave a blank line after the method signature. Check other methods for formatting consistency.